src/components/pages/download/index.svelte

```diff
@@ -128,4 +128,8 @@
     input {
         display: none;
     }
+
+    :global(.card:last-of-type) {
```

Comment:
this will change all cards globally even outside of the download page. if that's actually what you mean to do, you should move this rule to the Card component file